docs/pages/versions/v43.0.0/sdk/cellular.md

```diff
@@ -1,10 +1,13 @@
 ---
 title: Cellular
 sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-43/packages/expo-cellular'
+packageName: expo-cellular
```

Comment:
I'm puzzled why this is failing on this PR only

Reply:
`APIInstallSection` requires this property to be set via frontmatter. 

The pages on which you had to add this line does not use `APIInstallSection` component earlier.

Reply:
ah thanks for that explanation!